lighthouse-core/report/html/renderer/category-renderer.js

```diff
@@ -184,7 +184,7 @@ class CategoryRenderer {
       this.dom.find('.lh-category-header__description', tmpl).appendChild(descEl);
     }
 
-    return /** @type {Element} */ (tmpl.firstElementChild);
+    return tmpl;
```

Comment:
not a `querySelector` simplification, but you can `appendChild` a `DocumentFragment`. Not sure why we bothered with passing back the first child here